### PR TITLE
chore: deploy id automation

### PR DESCRIPTION
**Pull request by Skn0tt**
This PR adds a GH Actions workflow that opens a PR to update the Edge Bootstrap Deploy ID. I plan to either use a `deploy-succeeded` Netlify hook to call this workflow, or to call it from a GH workflow in edge-functions-bootstrap.

cc https://github.com/netlify/edge-bundler/pull/43#issuecomment-1153822651

### CODE REVIEW

File: .github/workflows/update-bootstrap-deploy-id.yml
Comment (eduardoboucas) [inline .github/workflows/update-bootstrap-deploy-id.yml:33]: Can we leave this up to the CODEOWNERS file?

**Review by EwanValentine — APPROVED** [review]
Nice idea!

File: .github/workflows/update-bootstrap-deploy-id.yml
Comment (Skn0tt) [reply .github/workflows/update-bootstrap-deploy-id.yml:33]: agreed, will be addressed in #45

**Pull request merged**
